spec/models/statement_spec.rb

```diff
@@ -14,6 +16,24 @@
     it { is_expected.to validate_uniqueness_of(:api_id).case_insensitive.with_message("API id already exists for another statement") }
   end
 
+  describe "scopes" do
+    describe ".with_state" do
+      it "selects only statements with states matching the provided name" do
+        expect(Statement.with_state("foo").to_sql).to include(%(WHERE "statements"."state" = 'foo'))
```

Comment:
I get that we do these tests elsewhere, but it might be worth us getting consistent on how we test scopes cc @cwrw

Reply:
agree, having a look at other models with scope tests they are testing which records are returned, can we do something similar here please?

Reply:
agree, we shouldn't be testing SQL